docs/developer/mesh.md

```diff
@@ -0,0 +1,270 @@
+# Mesh
+
+Here we describe the two mesh formarts that are used in RDycore. The two mesh
+formats that we use are:
+
+1. Exodus II
+2. DMPlex HDF5 v3.0.0
+
+RDycore has been tested for meshes that include triangular, quadrilateral, or
+both triangular and quadrilateral cells. The vertices of cells have coordinates
+in 3D.
+
+## Exodus II Mesh Format
+
+The Exodus II mesh format (`.exo`) uses 1-based indices.
+
+### Triangular elements
+
+```text
+     Triangular
+        cell
+         v3
+         / \
+        /   \
+      e5     e4
+     /   c1   \
+    /          \
+  v1 --- e3 --- v2
+```
+
+- The `elem_type` of the triangular cell, `c1`, is `TRI3`
+- It comprises of three vertices (i.e. `v1`, `v2`, `v3`)
+- It has five edges sets. The first two edge sets form a plane and the remaining
+  three edge sets are lines as follows:
+  - `e1`: Comprises of the plane formed by `v1`, `v2`, `v3` (not shown above)
+  - `e2`: Comprises of the plane formed by `v1`, `v3`, `v2` (not shown above)
```

Comment:
I think you are saying that `c1` is associated with five edges (edge sets are actually sets containing multiple edges in general). If that is true, I'm having trouble understanding what the edges `e1` and `e2` are. If I interpret your "plane formed by `v1`, `v2`, `v3` as the "(pseudo)vector formed by the cross product of `v1`, `v2`, `v3`", `e1` points "out of the screen", and correspondingly `e2` points "into the screen". But you're talking about planes and I'm talking about these pseudovectors. Is there a physical explanation of what the edges `e1` and `e2` represent?

Reply:
Now uses the term "_an oriented plane_"